+symbols index
+symbol find (<space>+f)
+fuzzy search
+build support
+git support
+location list (<space>+h/l)
+system clipboard (<space>+c/v)
--- 2022-10-24
+bitmap font
--- 2022-10-22
-tilda support
-remove modified bool (use text compare instead)
-full insert undo
-dot support
-do not up cursor when zz at beginning of file
--- 2022-10-21
-e-ink color scheme
-space whitespace mark
